src/main/java/edu/ie3/datamodel/io/factory/timeseries/CosmoIdCoordinateFactory.java

```diff
@@ -51,4 +52,9 @@ public String getLatField() {
   public String getLonField() {
     return LONG_GEO;
   }
+
+  @Override
+  public String getPointField() {
+    return Point;
+  }
```

Comment:
It seems like we only have the different coordinate sources to be able to parse and differentiate the `Cosmo` from the `Icon` coordinates. They are in so far different as they represent different coordinate grids which belong to the respective DWD model. `Icon` being their more international weather model and `Cosmo` a more regional one. Therefore we definitely need to make sure that we don't interchange them. Internally after initialization we only care about the `lat` and `lon` values and the id mapping. If we rather directly want a `Point` representation in the table which it seems like we do I do not see any reason to keep the `lat` and `lon` values as they only duplicate information.

Reply:
That's the direction that I thought of. So it'd make sense to have a third type for coordinate tables that use geometries, right? And then I'd make this type mandatory for SQL coordinates, unless we also want the (slower) implementation using lat/lon columns

Reply:
@staudtMarius After the discussion I had with @t-ober we came to the conclusion that creating a third coordinate type besides COSMO and ICON coordinates would be best. So that'd be a CoordinateFactory that only uses `id` and `point` (maybe rename it to `coordinate`) as fields.